### PR TITLE
Maintenance

### PR DESCRIPTION
**Pull request by timcadman**
> prerequisites:
> - install.packages("devtools")
> - install.packages("styler")
> - install.packages("pkgdown")

#### In general:

* [ ] Did you test the code against `https://armadillo.test.molgenis.org`?
* [x] Have you ran `styler:::style_active_pkg()` to style the code-files
* [x] Have you ran `devtools::check()` to be sure documentation is rerendered and code is quality checked?

#### Vignettes:

1. [ ] Did you create an .orig file and add it to the pre-render.R file?
2. [ ] Did you run the the pre-render.R file to regenerate the vignettes
3. [ ] Did you run `pkgdown::build_site()` to preview the documentation

### CODE REVIEW

File: R/meta-manual.R [hunk 1]
```diff
@@ -0,0 +1,23 @@
+dh.metaManual <- function(fit, method = "ML"){
```
Comment (sidohaakma) [inline R/meta-manual.R:1]: Is this an exported function? Anyway you need to describe it and use the @noRd annotation for internal functions.

File: R/pred-poly.R [hunk 2]
```diff
@@ -0,0 +1,172 @@
+
+dh.predPoly <- function(df, fixed, core_agevar, coeftab, conns, type = c("linear", "nonlinear")) {
```
Comment (sidohaakma) [inline R/pred-poly.R:2]: Same here

File: R/mean-by-occasion.R [hunk 3]
```diff
@@ -0,0 +1,196 @@
+dh.meanByOccasion <- function(df, agevar, outcome, bands, conns, grouping = NULL) {
```
Comment (sidohaakma) [inline R/mean-by-occasion.R:1]: Is this an exported function? Anyway you need to describe it and use the @nord annotation for internal functions.

File: R/make-lmer-form.R [hunk 4]
```diff
@@ -0,0 +1,27 @@
+dh.makeLmerForm <- function(
```
Comment (sidohaakma) [inline R/make-lmer-form.R:1]: Is this an exported function? Anyway you need to describe it and use the @nord annotation for internal functions.

File: R/make-age-polys.R [hunk 5]
```diff
@@ -0,0 +1,26 @@
+dh.makeAgePolys <- function(df, agevars, conns) {
```
Comment (sidohaakma) [inline R/make-age-polys.R:1]: Is this an exported function? Anyway you need to describe it and use the @nord annotation for internal functions.

File: R/lmer-tab.R [hunk 6]
```diff
@@ -0,0 +1,25 @@
+dh.lmerTab <- function(fit, coh, coh_format = "cols") {
```
Comment (sidohaakma) [inline R/lmer-tab.R:1]: Is this an exported function? Anyway you need to describe it and use the @nord annotation for internal functions.

**Review by sidohaakma — CHANGES_REQUESTED** [review]
Hi Tim, looks really nice. Keep the functions short approximately 300 lines is standard and always describe them even if they are internal functions. That way when you look at it later on you will know what it does. Use the @noRd annotation to

File: R/meta-manual.R (hunk 1 above)
Comment (timcadman) [reply R/meta-manual.R]: Exported as function

File: R/pred-poly.R (hunk 2 above)
Comment (timcadman) [reply R/pred-poly.R]: Exported as function

File: R/mean-by-occasion.R (hunk 3 above)
Comment (timcadman) [reply R/mean-by-occasion.R]: Exported as function

File: R/make-lmer-form.R (hunk 4 above)
Comment (timcadman) [reply R/make-lmer-form.R:21]: Exported as function

File: R/make-age-polys.R (hunk 5 above)
Comment (timcadman) [reply R/make-age-polys.R]: Exported as function

File: R/lmer-tab.R (hunk 6 above)
Comment (timcadman) [reply R/lmer-tab.R]: Exported as function

**Review by sidohaakma — CHANGES_REQUESTED** [review]
You need to style the package. Indentation is wrong at some points. Check the checklist ;-D.

**Comment by sidohaakma** [comment]
@timcadman you need to pull from the master as well, to resolve the conflicts.

**Review by sidohaakma — APPROVED** [review]

**Pull request merged**